concordia/language_model/local_models.py

```diff
@@ -0,0 +1,307 @@
+from concordia.language_model import language_model
```

Comment:
The pytype check is failing.
You'll need to add the following deps to setup.py:
-   langchain
-   torch
-   transformers
-   sentence_transfomers

Reply:
done

Reply:
not done: torch and transformers missing